beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -4586,16 +4595,25 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
             state.build_committee_cache(relative_epoch, &self.spec)?;
 
             let committee_cache = state.committee_cache(relative_epoch)?;
+            let committee_cache = Arc::new(committee_cache.clone());
```

Comment:
I think Clippy might complain about this clone?

```suggestion
            let committee_cache = Arc::new(committee_cache);
```

Reply:
This one is because the state gives us a reference to a committee cache. I had a peak at *taking* the committee cache, but it's a little tricky and I thought I'd keep the PR small. I'm open to trying it if you're keen for it.

Reply:
Oh right, nvm then. Shouldn't be too bad, I fear I'm micro-optimising here